Apps/W1/Shopify/app/src/Order handling/Tables/ShpfyOrderLineAttribute.Table.al

```diff
@@ -0,0 +1,40 @@
+/// <summary>
+/// Table Shpfy Order Line Attribute (ID 30123).
+/// </summary>
+table 30148 "Shpfy Order Line Attribute"
+{
+    Caption = 'Shopify Order Attributes';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "Order Id"; BigInteger)
+        {
+            Caption = 'Order Id';
+            DataClassification = CustomerContent;
+        }
+        field(2; "Order Line Id"; Guid)
+        {
+            Caption = 'Line Id';
+            DataClassification = CustomerContent;
+        }
+        field(3; "Key"; Text[100])
```

Comment:
Is there any documentation about the key length for custom attributes?
I'm unable to find any documentation on this topic for shopify.

Reply:
In Shopify it is documented as a string, https://shopify.dev/docs/api/storefront/2023-07/input-objects/AttributeInput
That's the reason why I used Copystr to fill the value in this field